src/library_sockfs.js

```diff
@@ -642,7 +642,7 @@ addToLibrary({
 #if SOCKET_DEBUG
           dbg(`websocket: queuing (${length} bytes): ${new Uint8Array(data)}`);
 #endif
-          dest.msg_send_queue.push(data);
+          dest?.msg_send_queue.push(data);
           return length;
```

Comment:
I'm not sure about this fix because it contradicts the comment maybe? "queue the message to be sent upon connect, and lie, saying the data was sent now.".  Here we are lying but not queueing.

Do you have a repro case for this?  Perhaps we can add a test case to test_sockets.py?

Reply:
Yeah, you're right. I'm gonna try to add a test case.